### PR TITLE
Change RPS to reflect reality

### PR DESCRIPTION
**Pull request by kulbida**

### CODE REVIEW

**Comment by bfrog** [comment]
I guess it all depends on your machine, I get 200K req/sec on mine with iron, a core i7 4770k

**Comment by reem** [comment]
Yeah, it really depends on your machine. I get roughly between 80 and 90, sometimes more. These are just microbenchmarks.

**Comment by kulbida** [comment]
sure it depends on the machine but I changed related page as well, the page where you can see a spec for the machine used in this benchmark. see https://github.com/iron/iron/wiki/How-to-Benchmark-hello.rs-Example
Probably it makes sense to display machine specific spec information in right next to the benchmark so all information will be in one place and it will not get out of sync? What do you think?

**Comment by kulbida** [comment]
I just don't understand, I'm getting 28+K at most and still have pretty powerful machine. @bfrog what machine you have?

**Comment by reem** [comment]
Yeah, seems reasonable to me. The stats in the repo are from running the benchmark on my machine:

MacBook Pro (Retina, 15-inch, Mid 2014)
Processor 2.8 GHz Intel Core i7
Memory 16 GB 1600 MHz DDR3

if you want to amend the PR to include that info, I would be happy to merge.

**Comment by bfrog** [comment]
@kulbida a desktop running arch linux with a core i7 4770k, 16gb ram, and a z87 chipset. Nothing too special. I think my ~2013 core i5 laptop gets ~100K rps, also linux. Might just be a mac thing?

**Comment by kulbida** [comment]
@reem ready for review.

**Comment by leodasvacas** [comment]
`[80+K` you have a typo there. `84,000+` was more precise so maybe you can keep that.

**Comment by kulbida** [comment]
@leodasvacas done!

File: README.md [hunk 1]
```diff
@@ -96,7 +96,17 @@ middleware. No plugins or middleware are bundled with Iron.
...
+  Model Name: MacBook Pro
+  Processor: Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
+  Memory (RAM): 16 GB
+  Total Number of Cores: 4
+  Total Number of Threads: 8
+  L2 Cache (per Core): 256 KB
+  L3 Cache: 6 MB
+```
+See details here https://github.com/iron/iron/wiki/How-to-Benchmark-hello.rs-Example
 and is mostly IO-bound, spending over 70% of its time in the kernel send-ing or
```
Comment (reem) [inline README.md]: this line now starts in the middle of a sentence, can you massage the english here a bit?

File: README.md [hunk 2]
```diff
@@ -96,7 +96,17 @@ middleware. No plugins or middleware are bundled with Iron.
 
 ## Performance
 
-Iron averages [84,000+ requests per second for hello world](https://github.com/iron/iron/wiki/How-to-Benchmark-hello.rs-Example)
+Iron averages [84,000+ requests per second. We run the benchmark test using:
```
Comment (untitaker) [inline README.md]: You still have the stray bracket here.

**Comment by reem** [comment]
Closing due to inactivity, feel free to remake the PR with the requested changes.

**Pull request closed without merging**